mscore/scoreview.cpp

```diff
@@ -110,7 +110,7 @@ ScoreView::ScoreView(QWidget* parent)
       setAttribute(Qt::WA_OpaquePaintEvent);
 #endif
       setAttribute(Qt::WA_NoSystemBackground);
-      setFocusPolicy(Qt::ClickFocus);
+      setFocusPolicy(Qt::StrongFocus);
```

Comment:
Aside from change to skip the updateInspector() if not visible, and the attempt to eliminate an extra endCmd(), this here is the only other change that seems like it could have any effect on anything but screenreader behavior.  Basically, it's saying that it's OK for the widget to take focus by keyboard, which indeed it should be.